assets/js/listeners.js

```diff
@@ -35,30 +35,23 @@ window.addEventListener("moon:update-accordion-aria", (event) => {
   }
 });
 
-window["moon:breadcumbs-close-handlers"] = {};
-
-window.addEventListener("moon:show-collapsed-breadcrumbs", (event) => {
+window.addEventListener("moon:toggle-collapsed-breadcrumbs", (event) => {
```

Comment:
are old breadcrumbs (v1) still working? we have either keep previous events also, or (preffered) - make old breadcrumbs use this listeners also